root/components/relationship-editor.tt

```diff
@@ -47,11 +47,36 @@
     </div>
     <div>
       <label>
-        <input type="checkbox" data-bind="checked: $parent.changeAllRelationshipCredits" />
-        <span data-bind="html: MB.i18n.l('Change all relationship credits for {entity} on the page.', {entity: entity.html({target: '_blank'})})">
+        <input type="checkbox" data-bind="checked: $parent.changeOtherRelationshipCredits[role]" />
+        <span data-bind="html: MB.i18n.l('Change credits for other {entity} relationships on the page.', {entity: entity.html()})">
         </span>
       </label>
     </div>
+    <div data-bind="visible: $parent.changeOtherRelationshipCredits[role]">
+      <div>
+        <label>
+          <input type="radio" value="all" data-bind="checked: $parent.selectedRelationshipCredits[role]" />
+          [% l('All of these relationships.'); %]
+        </label>
+      </div>
+      <div>
+        <label>
+          <input type="radio" value="same-entity-types" data-bind="checked: $parent.selectedRelationshipCredits[role]" />
+          <span data-bind="html: MB.i18n.l('Only relationships to {entity_type} entities.',
+                                           {entity_type: MB.i18n.strings.entityName[relationship.target(entity).entityType]})">
+          </span>
+        </label>
+      </div>
+      <div>
+        <label>
+          <input type="radio" value="same-relationship-type" data-bind="checked: $parent.selectedRelationshipCredits[role]" />
+          <span data-bind="html: MB.i18n.l('Only “{relationship_type}” relationships to {entity_type} entities.',
+                                           {relationship_type: $parent.linkTypeName(),
+                                            entity_type: MB.i18n.strings.entityName[relationship.target(entity).entityType]})">
```

Comment:
Called `toLocaleLowerCase()` for entity type in https://github.com/metabrainz/musicbrainz-server/pull/488/commits/8c853861912e9eed2f589c70ff157b63d7993219#diff-dbe28ef3e31e909f5576de5c67298782R339.